tests/test_keys.py

```
@@ -11,39 +11,39 @@
 
 @pytest.fixture
 def _mock_graphql() -> Generator[AsyncMock, None, None]:
-    with patch("unraid_mcp.tools.keys.make_graphql_request", new_callable=AsyncMock) as mock:
+    with patch("unraid_mcp.tools.unraid.make_graphql_request", new_callable=AsyncMock) as mock:
         yield mock
 
 
 def _make_tool():
```

Ruff (0.15.5)

[warning] Missing return type annotation for private function `_make_tool`

(ANN202)